DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/src/Beamformer.hpp

```diff
@@ -75,16 +75,12 @@ event SubmitBeamformerKernel(
 
   auto e = q.submit([&](handler& h) {
     h.single_task<BeamformerKernelName>([=] {
-      // count the number of xrx vectors that have been processed with the
-      // current weight vectors
-      int xrx_vectors_processed = 0;
-
       while (1) {
         CalcType weight_vectors[k_num_weight_vectors][kNumCalcTypePerVector];
 
         // load the weight vectors to be used with the next set of Xrx vectors
-        for (char vector_num = 0; vector_num < (char)k_num_weight_vectors;
-             vector_num++) {
+        for (unsigned char vector_num = 0;
+             vector_num < (char)k_num_weight_vectors; vector_num++) {
```

Comment:
The convention for k_num_weight_vectors seems strange.  Should be either kNumWeightVectors (if arbitrary "magic" constant) or num_weight_vectors.

Reply:
I was told that template arguments in code samples must follow the naming convention of variables: all lower case separated by underscores.  I have adopted the convention to use k_ at the start of template variables that are treated as compile time constants.  I would have preferred to name my template variables with the same format as constants (kNumWeightVectors instead of k_num_weight_vectors) but was told this had been explicitly disallowed in previous code sample submissions.